imgboard.php

```diff
@@ -19,7 +19,8 @@
 
 */
 
-require "config.php";
+require("config.php");
+require("config.json");
```

Comment:
What's the point of a loader class then?
And as it stands we'll have to global in the json vars (after decoding them into `$config`) or something.
A ***lot** of code that worked purely off constants now won't.

Apparently PHP 5.6+ supports constant arrays, but you're playing with fire.